.ibm/pipelines/kubernetes-tests.sh

```diff
@@ -12,6 +12,7 @@ cleanup_namespaces
 
 (
     set -e
+    make goget-ginkgo
```

Comment:
Since we are still running Ginkgo from the `vendor` folder, maybe this is no longer necessary.

Reply:
removed